modules/proxyconnection/ProxyConnectionService.js

```diff
@@ -34,12 +37,21 @@ export default class ProxyConnectionService {
      * jid to send the message to and the message
      */
     constructor(options = {}) {
+        const {
+            jitsiConnection,
+            ...otherOptions
+        } = options;
+
         /**
          * Holds a reference to the collection of all callbacks.
          *
          * @type {Object}
          */
-        this._options = options;
+        this._options = {
+            iceConfig: jitsiConnection
+                && jitsiConnection.xmpp.connection.jingle.p2pIceConfig,
+            ...otherOptions
+        };
```

Comment:
We'll handle refreshing some other day, yeah?

Reply:
Also, are we guaranteed to have p2pIceConfig defined?

Reply:
Yes, I was thinking to do that in the xmpp module. In spot they will be fetched each time new screensharing connection is created ?

Reply:
we don't guarantee, but then it should give `iceConfig` `undefined` and then it should default to `DEFAULT_STUN_SERVERS` as far as I understand